src/textual/_border.py

```diff
@@ -15,6 +15,7 @@
 OUTER = 2
 
 BORDER_CHARS: dict[EdgeType, tuple[str, str, str]] = {
+    # TODO: in "browsers' CSS" `none` and `hidden` both set the border width to zero. Should we do the same?
```

Comment:
Just because as a Web developer I was surprised to see that in Textual borders that are set to `hidden` or `none` still have a width, which contrasts to the _"browsers' CSS"_ behaviour.
> Like the hidden keyword, `none` displays no border. Unless a background-image is set, the computed value of the same side's border-width will be 0, even if the specified value is something else. 

But I know that our primary target is _not_ Web developers like myself, so we might want to keep that behaviour as is of course :slightly_smiling_face:

Reply:
No, you're probably right. It should match the browser. I would like to have something that reserved space but didn't render anything. Perhaps "blank"?